documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -379,74 +501,124 @@ To alter the limits along the x, y, or z axes you use the commands ``lx, ly`` an
 
 
 to change the limits along x to be 0 and 2, and so on.
-To change the axes to log-scale, you use the commands ``logx, logy`` and ``logz``, and to change to a linear scale you use ``linx``\ ...etc. A full list of formatting options can be found `here <http://www.libisis.org/User_Manual#Plot_Commands>`__.
+
+To change the axes to log-scale, you use the commands ``logx``, ``logy`` and ``logz``, and to change to a linear scale you
+use ``linx``\ ...etc. A full list of formatting options can be found `here
+<http://www.libisis.org/User_Manual#Plot_Commands>`__.
 
 Fitting
 =======
 
-You can also use Horace to fit your data. It can take quite a long time for the fit to converge, so it is therefore a good idea to provide a good initial guess of the fit parameters. You can work these out simulating and then comparing the result to the data by eye.
+You can also use Horace to fit your data. It can take quite a long time for the fit to converge, so it is therefore a
```

Comment:
Is it obvious what "fitting" is? Maybe say "use Horace to fit a theoretical prediction to the experimental data by changing its parameters"